client/api/assessments.js

```diff
@@ -0,0 +1,12 @@
+import request from 'superagent'
+
+const URL = 'http:localhost:3000/v1'
```

Comment:
Please have this as `http:localhost:3000/api/v1/assessment` as others will need to use this as well in this file.